.gitignore

```diff
@@ -6,6 +6,7 @@
 /captures
 .externalNativeBuild
 /site
+.idea/
```

Comment:
I appreciate having `.idea` files in the repo is annoying, however, not all of them are useless. e.g. copyright templates.